package.json

```diff
@@ -43,7 +43,9 @@
     "test:site": "npm run clean:site && ./scripts/site-generator-test.sh"
   },
   "devDependencies": {
+    "@babel/core": "^7.4.5",
```

Comment:
I don't see you using this anywhere? Is core necessary?

Reply:
This is required by https://github.com/babel/karma-babel-preprocessor